.github/workflows/s3-deploy.yml

```diff
@@ -0,0 +1,35 @@
+name: Production Build Deployment
+on:
+  push:
+    branches:
+      - master
+      - y
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Install dependencies
+        run: |
+          npm install
+      - name: Run unit tests
+        run: |
+          CI=true npm test --watch=all
```

Comment:
Since there are no unit tests in the repo, this simply verifies that the bundle renders without crashing.

Reply:
@arberx looks good! Are you able to add a simple unit test so we can make sure not to auto-deploy code that doesn't render?

Reply:
There already is a [test](https://github.com/iearn-finance/iearn-finance/blob/y/src/App.test.js) in the repo that tests a render. 

This is provided by default when you use `create-react-app` I believe.